src/main/resources/com/mathworks/ci/MatlabBuilder/help-matlabRoot.html

```diff
@@ -1,5 +1,7 @@
 <div>
-  Enter the full path to the MATLAB root folder, which is returned by the <i><b>matlabroot</b></i> function. <br><br>Example:<br><br><i><b><u>Windows:</u><b></i><br> C:\Program Files\MATLAB\R2019a <br>
+  Enter the full path to the MATLAB root folder, which is returned by the <i><b>matlabroot</b></i> function. <br><br>Example:<br><br><i><b><u>Windows:</u></b></i><br> C:\Program Files\MATLAB\R2019a <br>
   <i><b><u>Linux:</u></b></i><br> /usr/local/MATLAB/R2019a <br>
   <i><b><u>Mac:</u></b></i><br> /Applications/MATLAB_R2019a.app <br>
+  <p>
+  <b>Note:</b> <mark>If the job is running on remote agents then, Full path of MATLAB root folder on the specific remote agent should be entered.</p></mark>
```

Comment:
Please change the note as follows.

<b>Note:</b> If the job runs on a remote agent, you must specify the full path to MATLAB root folder on the remote agent.

Reply:
Done.